Update index.md (#40)
mParticle/docs: src/pages/integrations/kochava/event/index.md

> mParticle will forward all events generated by mParticle's `logEvent` SDK method will be forwarded to Kochava, with the mParticle Event Name as the Kochava Event Title.  See the panel below for a sample call using Kochava's SDK, and the equivalent using mParticle's.

Reviewer: while you're here, can you fix this typo? 

```suggestion
mParticle will forward all ["custom" events](https://docs.mparticle.com/developers/server/json-reference/#custom_event) to Kochava, using the mParticle event name as the Kochava event "title."  See the panel below for a sample call using Kochava's SDK, and the equivalent using mParticle's.
```

> In contrast, Application State Transition events will only be forwarded to Kochava if they meet the criteria of an install event ("type" = "appInit" and "isFirstRun" = "true").

Reviewer: ```suggestion
In contrast, ["Application State Transition" events](https://docs.mparticle.com/developers/server/json-reference/#application_state_transition) will only be forwarded to Kochava if they meet the criteria of an install event:("application_transition_type" = "application_initialized" and "is_first_run" = "true").
```
